dev-shared.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
api's `dev.sh` uses `gunicorn --reload` which already gives us hot-reloading. ideally we don't need to add another layer with `watchmedo` here

however, `gunicorn --reload` doesn't appear to have a way to ignore files matching a pattern like `watchmedo` does which is a bummer

because this is just in our dev environment, we could possibly try moving `dev.sh` to use `uvicorn` instead which has `--reload-exclude` https://www.uvicorn.org/#command-line-options

(i googled the difference: `gunicorn` is a WSGI server which means it processes requests serially and `uvicorn` is an ASGI server which means it can process requests concurrently.)

alternatively, a hot-reload honestly isn't that disruptive, if we run it for shared no matter what it wouldn't be the worst thing in the world

in any case, i think we can add what we want to `dev.sh` and skip adding this second file
```bash
if [ "$EDITABLE_SHARED" = "y" ]
then
    pip install -e ./shared
    uvicorn blah blah blah
else
    gunicorn blah blah blah
fi
```